person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] Please use the `random` library to generate the random number for the ID of the newly added person automatically, adding the ID, which is not defined and has the `nil` property is not going to help you maintain the record, at on later stage you will add the functionality based on the id. So that's why it is essential to generate the ID implicitly instead of adding it explicitly.